lib/user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr_reader :name, :joke
+
+  def initialize(name, joke)
+    @name = name
+    @joke = []
+  end
+
+  def user
```

Comment:
This is an unnecessary method, and could be dangerous - if any one calls `.user` on a User object, it will change that user's name to 'Sal'!  It would look like this: 

```
megan = User.new('Megan')
=> <#User034234......>
megan.name
=> 'Megan'
megan.user
=> 'Sal'
megan.name
=> 'Sal' 
```

Try not to create methods that you don't need!